app/src/main/java/org/kinecosystem/kinit/network/Wallet.kt

```diff
@@ -33,24 +33,26 @@ private const val MAIN_NET_WALLET_CACHE_NAME = "kin.app.wallet.mainnet"
 
 private const val NETWORK_ID_MAIN = "Public Global Kin Ecosystem Network ; June 2018"
 private const val NETWORK_ID_TEST = "Kin Playground Network ; June 2018"
-
+private const val KIN_ISSUER_MAIN = "GDF42M3IPERQCBLWFEZKQRK77JQ65SCKTU3CW36HZVCX7XX5A5QXZIVK"
```

Comment:
why is it not incapsulated in the  service provider ?

Reply:
because it depends on the Build Type